### ISSUE TITLE
Generating a list of pages in a category

### ISSUE THREAD

**Issue by jarble** [valerio-bozzolan/CategoryToolbox#1]
Is it possible to generate a list of all pages in a category using the CategoryToolbox extension?
I want to generate a list of pages to use a in Lua module, but I don't know if this extension has this feature.

**Comment by valerio-bozzolan** [comment]
Yep this was not really an intended feature but the extension actually exposes the ` mw.ext.cattools.categoryPages()` entry point useful to return the first `25` elements in a category.

https://github.com/valerio-bozzolan/CategoryToolbox/blob/master/includes/CategoryToolbox_LuaLibrary.php#L74-L92
